config/initializers/reserved_words.rb

```diff
@@ -46,7 +46,6 @@ class ReservedWords
     dashboard
     day
     deep
-    delayed_job_admin
```

Comment:
I would keep this, since it's a reserved word 👀 It'd be weird to have a dev.to/delayed_job_admin user anway :D